apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManager.java

```diff
@@ -165,12 +166,20 @@ public static AbstractSpan activeSpan() {
         return get().activeSpan();
     }
 
+    /**
+    * Plese use `ContextManager.stopSpan(span)` if could, it's can "fail-fast"
```

Comment:
```suggestion
    * Recommend use ContextManager::stopSpan(AbstractSpan span), because in that way, 
    * the TracingContext core could verify this span is the active one, in order to avoid stop unexpected span.
```

Reply:
I hope you accept this comment change suggestion.